src/include/kdump/uifunctions.rb

```diff
@@ -1125,7 +1127,7 @@ def InitEnableReboot(_key)
       UI.ChangeWidget(
         Id("EnableReboot"),
         :Value,
-        Ops.get(Kdump.KDUMP_SETTINGS, "KDUMP_IMMEDIATE_REBOOT") == "yes" ? true : false
+        Set["yes", "true", "1"] === Ops.get(Kdump.KDUMP_SETTINGS, "KDUMP_IMMEDIATE_REBOOT") ? true : false
```

Comment:
this is not common way in ruby. It is usually array include
```suggestion
        ["yes", "true", "1"].include?(Ops.get(Kdump.KDUMP_SETTINGS, "KDUMP_IMMEDIATE_REBOOT"))
```
as advantage result of include? is already boolean so that ternary operator is not needed.